common/list/_lists.scss

```
@@ -38,18 +41,13 @@
 /* Double space list items
 	Might worth to publish it upstream here instead: https://github.com/wet-boew/wet-boew/blob/master/src/base/lists/_base.scss#L30
  */
-
+div.lst-spcd-2>ul,
```

Be as specific the current one. I don't see why we would like to explicitly limit this to `<ul>` only. Make it more generic or include the `<ol>` in scope

```suggestion
div.lst-spcd-2 > ul,
div.lst-spcd-2 > ol,
```

---

Done

---

I don't see this update.